app/src/services/market_maker.ts

```diff
@@ -84,6 +84,7 @@ class MarketMakerService {
     try {
       const overrides = {
         value: '0x0',
+        gasLimit: 750000,
```

Comment:
Please notice this, If we don't add this gasLimit the add funding transaction will fail